src/algebra/archimedean.lean

```diff
@@ -150,6 +150,9 @@ archimedean_iff_rat_lt.trans
  λ H x, let ⟨n, h⟩ := H x in ⟨n+1,
    lt_of_le_of_lt h (rat.cast_lt.2 (lt_add_one _))⟩⟩
 
+/-- `round` rounds a number to the nearest integer. `round (1 / 2) = 1` -/
+def round [floor_ring α] (x : α) : ℤ := ⌊x + 1 / 2⌋
```

Comment:
Probably `round` should go at the end of `algebra.floor`, along with `abs_sub_round` and `rat.cast_round` which don't need an archimedean field either.

Reply:
Done.